hoot-core-test/src/test/cpp/hoot/core/OsmMapTest.cpp

```diff
@@ -32,43 +32,38 @@
 #include <cppunit/TestFixture.h>
 
 // Hoot
-#include <hoot/core/conflate/Conflator.h>
-#include <hoot/core/util/MapProjector.h>
 #include <hoot/core/OsmMap.h>
+#include <hoot/core/TestUtils.h>
```

Comment:
This is where the `HootTestFixture` class resides.  It is added to almost every file I changed.